apps/site/src/app/apply/sections/Form/Form.tsx

```diff
@@ -41,14 +39,14 @@ export default function Form() {
 			}
 		}
 
-		const formEntries = Object.fromEntries(formData.entries());
-		console.debug(formEntries);
+		// const formEntries = Object.fromEntries(formData.entries());
+		// console.debug(formEntries);
 
 		try {
 			const res = await axios.post(APPLY_PATH, formData);
 			if (res.status === 201) {
 				console.log("Application submitted");
-				router.push("/portal");
+				window.location.href = "/portal";
```

Comment:
Could we include a comment indicating why we use `window.location` instead of `router.push`?